release/_releases_json.sh

```diff
@@ -317,6 +317,30 @@ function _promote_product_versions {
 	done
 }
 
+function _tag_jakarta_product_versions {
+	local product_version_json_file
+
+	find "${_PROMOTION_DIR}" -maxdepth 1 -type f -name "*.json" | while read -r product_version_json_file
+	do
+		jq 'map(
+				(.productGroupVersion | ascii_upcase) as $current_q |
```

Comment:
@GabrielFernandoS, can you please remove the `ascii_upcase`?

Also, please rename `current_q` to `currentProductGroupVersion`